cli/_posts/2017-01-01-how-to-use-the-runnable-c-l-i.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: How to use the Runnable CLI
+title: How To Use The Runnable CLI
```

Comment:
Good catch on capitalizing "Use", but "to" and "the" should still be lowercase. I use http://titlecase.com/ whenever I'm not certain.